parallax/config.py

```diff
@@ -10,8 +10,8 @@
         "show_axes": True,
         "auto_select_corr_points": True,
     },
-    "calibration_path": "./calibrations",
-    "console_history_file": "./console_history",
+    "calibration_path": "./",
```

Comment:
Do you prefer calibrations here, or was tere another reason to make this change?